marklogic-data-hub/src/main/java/com/marklogic/hub/connector/Writer/MarkLogicDataWriter.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.hub.connector.Writer;
```

Comment:
For the package, let's do com.marklogic.hub.cloud.aws.glue - I know that's long, but it's consistent with an existing package we have in the Hub Central project. "com.marklogic.hub" has to be there, and "cloud" is the parent package for the eventual "azure" subpackage. And then "glue" because we may have additional things specific to AWS that don't involve glue. I am guessing we do not need an additional "connector" subpackage, as anything in the "glue" package would necessarily be associated with the connector. Does that seem right?